client/components/card-heading/index.jsx

```diff
@@ -18,7 +17,7 @@ function CardHeading( { tagName = 'h1', size = 20, isBold = false, className, id
 		className && className,
 		classNameObject
 	);
-	return createElement( tagName, { className: classes, id }, preventWidows( children, 2 ) );
```

Comment:
As far as I could find, `CardHeading` is always passed a heading tag as `tagName`. It might make sense to enforce that usage via types.